Conceptpower+Spring/src/main/webapp/WEB-INF/views/auth/concepts/ConceptWrapperAddView.jsp

```diff
@@ -1,608 +1,554 @@
-										value="${concept.entry.word}"></c:out></a></font>
-						</td>
-						<td align="justify"><c:out value="${concept.entry.id}"></c:out></td>
-						<td align="justify"><c:out value="${concept.entry.wordnetId}"></c:out></td>
-						<td align="justify"><font size="2"><c:out
-									value="${concept.entry.pos}"></c:out></font></td>
-						<td align="justify"><font size="2"><c:out
-									value="${concept.entry.conceptList}"></c:out></font></td>
-						<td align="justify"><font size="2">
-								${concept.description}</font></td>
-						<td align="justify"><font size="2"><c:out
-									value="${concept.type.typeName}"></c:out></font></td>
-						<td align="justify"><font size="2"><c:forEach
-									var="syn" items="${concept.synonyms}">
-									<c:out value="-> ${syn.word}"></c:out>
-								</c:forEach></font></td>
-					</tr>
-				</c:forEach>
-			</tbody>
-		</table>
-
-	</c:if>
-
-	<p />
-	<div id="selectedconcepts"></div>
-
-	<h2>3. Enter additional information</h2>
-	<p>Selected Wordnet concept:</p>
-
-	<table class="table table-striped table-bordered">
-
-		<tr>
-			<td>Concept List</td>
-
-			<td><form:select path="lists" name="lists" class="form-control">
-					<form:option value="" label="Select concept list" />
-					<form:options items="${lists}" />
-				</form:select></td>
-			<td />
-
-		</tr>
-		<tr>
-			<td>Description</td>
-			<td><input type="text" name="description" id="description"></td>
-			<td />
-		</tr>
-		<tr>
-			<td>Synonyms</td>
-			<td><div id="addedSynonyms"></div></td>
-			<td width="50"><input type="button" name="synonym" id="addsynonym"
-				data-toggle="modal" data-target="#synonymModal" value="Add Synonym"
-				class="btn btn-primary"></td>
-		</tr>
-		<tr>
-			<td>Concept Type</td>
-			<td><form:select path="types" name="types" class="form-control">
-					<form:option value="" label="Select one" />
-					<form:options items="${types}" />
-				</form:select></td>
-			<td />
-		</tr>
-		<tr>
-			<td>Equals</td>
-			<td><input type="text" name="equals" id="equals"></td>
-			<td />
-		</tr>
-		<tr>
-			<td>Similar to</td>
-			<td><input type="text" name="similar" id="similar"></td>
-			<td />
-		</tr>
-		<tr hidden="true">
-			<td><input type="text" name="synonymsids" id="synonymsids"></td>
-		</tr>
-		<tr hidden="true">
-			<td><input type="text" name="wrapperids" id="wrapperids"></td>
-		</tr>
-
-	</table>
-	<br /> <input type="submit" disabled="disabled" id="createwrapper"
-		value="Create Wordnet concept wrapper" class="btn btn-primary">
+    action="${pageContext.servletContext.contextPath}/auth/conceptlist/addconceptwrapper/add"
+    method='post'>
+    <h2>2. Select Wordnet concept from search results</h2>
+
+    <h4>The following concepts were found:</h4>
+    Remember, you can only create a wrapper for concepts in Wordnet!
+    <p></p>
+
+    <c:if test="${not empty result}">
+        <table cellpadding="0" cellspacing="0"
+            class="table table-striped table-bordered"
```

Comment:
Removed table from the class attribute. Table has been removed to make screen look better when text area is added.